src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -368,6 +375,76 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 5,
                 "Current action does not explicitly specify any content types for the response.");
+
+            _selectingOutputFormatterUsingAcceptHeader = LoggerMessage.Define(
+                LogLevel.Debug,
+                6,
+                "Attempting to select an output formatter based on Accept header.");
+
+            _selectingOutputFormatterWithoutUsingAcceptHeader = LoggerMessage.Define(
+                LogLevel.Debug,
+                7,
+                "Attempting to select an output formatter without using Accept header.");
```

Comment:
We should also include the content types that were specified on the result so a developer can see why.